docs/local-setup/running-testnet.md

```diff
@@ -141,18 +134,25 @@ sudo apt install -y git binutils-dev libcurl4-openssl-dev zlib1g-dev libdw-dev l
 
 ## Success Message
 
-Once you have followed either of the above steps, you should see a success message that looks similar to either of the messages below:
+Once you have followed the steps for running a node with Docker or of Compiling without Docker, you should see messages similar to as shown below:
+
 
-Creating genesis
-**Genesis created**
-Starting NEAR client and Watchtower dockers...
+```text
+Using local binary at path/to/nearcore/target/{debug, release}
+Our genesis version is up to date
+Starting NEAR client...
+Node is running! 
+To check logs call: `nearup logs` or `nearup logs --follow`
+```
 
-**Node is running!**
-[2019-10-28T20:37:11Z INFO near_network::peer_manager] 
+or
 
-**Server listening at** ed25519:HbhhYv3J87NBzMGzZdEEMKGjXoDQr7sb1Y5skfiBYvZT @ 0.0.0.0 : 24567
+```text
+Using local binary at path/to/nearcore/target/{debug, release}
```

Comment:
same as above, may be ok but better to be consistent

Reply:
removed braces from all the commands